management/src/main/java/io/micronaut/management/endpoint/health/HealthEndpointSelector.java

```diff
@@ -0,0 +1,22 @@
+package io.micronaut.management.endpoint.health;
+
+/**
+ * Options for {@link io.micronaut.management.endpoint.health.HealthEndpoint} selector that aggregates
+ * {@link io.micronaut.management.health.indicator.HealthIndicator} according to {@link io.micronaut.management.health.indicator.Liveness}
+ * respectively {@link io.micronaut.management.health.indicator.Readiness} qualifiers.
+ *
+ * @author Pavol Gressa
+ * @since 2.1
+ */
+public enum HealthEndpointSelector {
```

Comment:
I would move this to the package `io.micronaut.management.health.indicator` and rename to `HealthCheckType`

Reply:
OK